### PR TITLE
Make struct comment much clearer.

### PR DESCRIPTION
**Pull request by emacsliu1107**
**What this PR does / why we need it**:
Did some comments.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/proxy/endpoints.go [hunk 1]
```diff
@@ -83,14 +83,15 @@ type makeEndpointFunc func(info *BaseEndpointInfo) Endpoint
 type EndpointChangeTracker struct {
 	// lock protects items.
 	lock sync.Mutex
-	// hostname is the host where kube-proxy is running.
+	// hostname is the host where kube-proxy is running on.
 	hostname string
-	// items maps a service to is endpointsChange.
+	// items maps a service to a endpointsChange.
```
Comment (neolit123) [inline pkg/proxy/endpoints.go]: `a` -> `an`

File: pkg/proxy/endpoints.go [hunk 2]
```diff
@@ -83,14 +83,15 @@ type makeEndpointFunc func(info *BaseEndpointInfo) Endpoint
 type EndpointChangeTracker struct {
 	// lock protects items.
 	lock sync.Mutex
-	// hostname is the host where kube-proxy is running.
+	// hostname is the host where kube-proxy is running on.
```
Comment (neolit123) [inline pkg/proxy/endpoints.go]: not really needed, but still valid.

**Comment by ShualLiu** [comment]
why 'pull-kubernetes-verify' job failed? I can't figure out after I saw the build log. @idealhack

**Comment by idealhack** [comment]
@ShualLiu The error log shows `gofmt` failed.

File: pkg/proxy/endpoints.go [hunk 3]
```diff
@@ -83,14 +83,15 @@ type makeEndpointFunc func(info *BaseEndpointInfo) Endpoint
...
 	hostname string
-	// items maps a service to is endpointsChange.
+	// items maps a service to an endpointsChange.
 	items map[types.NamespacedName]*endpointsChange
 	// makeEndpointInfo allows proxier to inject customized information when processing endpoint.
 	makeEndpointInfo makeEndpointFunc
 	// isIPv6Mode indicates if change tracker is under IPv6/IPv4 mode. Nil means not applicable.
 	isIPv6Mode *bool
+	// recorder records events.
 	recorder   record.EventRecorder
```
Comment (idealhack) [inline pkg/proxy/endpoints.go]: Please remove two spaces here.

**Comment by idealhack** [comment]
Squash commits to one after you fix it, thanks!

**Pull request closed without merging**